src/style.css

```
@@ -1 +1,46 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image:url(images/star.svg)
+}
+
+.stars__star:nth-last-child(1) {
+  margin-right: 0;
+}
+
+.stars--1 > .stars__star:nth-of-type(-n+1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 > .stars__star:nth-of-type(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 > .stars__star:nth-of-type(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 > .stars__star:nth-of-type(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 > .stars__star:nth-of-type(-n+5) {
+  background-image: url(images/star-active.svg);
+}
```

you can combine it :)
```suggestion
.stars--1 > .stars__star:nth-of-type(-n+1),
.stars--2 > .stars__star:nth-of-type(-n+2),
.stars--3 > .stars__star:nth-of-type(-n+3),
.stars--4 > .stars__star:nth-of-type(-n+4),
.stars--5 > .stars__star:nth-of-type(-n+5) {
  background-image: url(images/star-active.svg);
}
```